src/main/java/uk/gov/hmcts/reform/unspec/service/robotics/RoboticsNotificationService.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.unspec.service.robotics;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sendgrid.EmailData;
+import uk.gov.hmcts.reform.sendgrid.SendGridClient;
+import uk.gov.hmcts.reform.unspec.config.properties.robotics.RoboticsEmailConfiguration;
+import uk.gov.hmcts.reform.unspec.model.CaseData;
+import uk.gov.hmcts.reform.unspec.model.robotics.RoboticsCaseData;
+import uk.gov.hmcts.reform.unspec.service.robotics.mapper.RoboticsDataMapper;
+
+import javax.validation.constraints.NotNull;
+
+import static java.util.List.of;
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.sendgrid.EmailAttachment.json;
+
+@Slf4j
+@Service
+@AllArgsConstructor
+public class RoboticsNotificationService {
+
+    private final SendGridClient sendGridClient;
+    private final RoboticsEmailConfiguration roboticsEmailConfiguration;
+    private final RoboticsDataMapper roboticsDataMapper;
+    private final ObjectMapper objectMapper;
+
+    public void notifyRobotics(@NotNull CaseData caseData) {
```

Comment:
We can change this later but I think for contract testing bit we discussed we will need to allow specifying the email subject here so that we can include a reference of the contract the email is sent part of.